src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(aim) {
+    if (aim instanceof Herbivore && !aim.hidden) {
+      aim.health -= 50;
+
+      if (aim.health === 0) {
+        const result = Animal.alive.filter(x => x.health > 0);
```

Comment:
```suggestion
        const liveAnimals = Animal.alive.filter(animal => animal.health > 0);
```
Let's use more meaningful names for your variables